src/utils/api.js

```diff
@@ -27,3 +27,9 @@ export const voteArticle = (article_id) => {
     return res.data;
   });
 };
+
+export const postComment = (article_id, comment) => {
+  return AgoraApi.post(`/articles/${article_id}/comments`, comment).then((res) => {
```

Comment:
I'd extract all of the API requests into the API.js, rather than importing in your baseURL made with axios create, and making the api request here. Essentially create functions in you API.js that abstract all the stuff away that interacts with your API.